rasa/__main__.py

```diff
@@ -93,7 +93,7 @@ def print_version() -> None:
 
 
 def main() -> None:
-    # Running as standalone python application
+    """Running as standalone python application"""
```

Comment:
```suggestion
    """Run as standalone python application"""
```

By convention we use the imperative